articles/cloud-storage/cs-how-python-boto-ecs.md

```diff
@@ -0,0 +1,160 @@
+---
+title: How to interact with ECS using Python and Boto| UKCloud Ltd
+description: How to use Python and Boto with ECS
+services: cloud-storage
+author: Paul Brown
+toc_rootlink: How To
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: How to use Python and Boto with ECS
+toc_fullpath: How To/cs-how-python-boto-ecs.md
+toc_mdlink: cs-how-python-boto-ecs.md
+---
+
+# How to use Python and Boto with ECS
+
+## What is Boto?
+
+Boto is a Python package which enables interaction with UKCloud's Cloud Storage and will run on any client supporting Python.
+
+> [!NOTE]
+> This guide uses Python 3 and Boto 3 and has not been tested on previous versions.
+
+## Use cases
+
+You can use Boto to:
+
+- Automate Cloud Storage interactions, for example the creation and deletion of buckets, the uploading, downloading and deletion of objects.
+
+- Use Cloud Storage as a target for backups or long-term file retention.
+
+## Installing Python and Boto
+
+1. Navigate to the [Python homepage](https://www.python.org/) and install the version of Python 3 relevant for your OS.
+
+2. Next install the Boto 3 package via Python's Package Manager,  Pip.
+
+```Python
```

Comment:
I think you'll need to indent this and the next two lines by four spaces so that the numbered steps render correctly